### DISCUSSION TITLE
TEXTURED MESH SIMPLIFICATION

### DISCUSSION THREAD

**Discussion by JCAELA** [cnr-isti-vclab/PyMeshLab#212] [Q&A] [answered]
Hi all,

I am trying to reduce the size of a textured mesh via "simplification_quadric_edge_collapse_decimation_with_texture()" filter. Unfortunately, I do not know to insert the arguments of this filter.

Do you have any simple example of a script where this filter is applied?

Thank you

**Comment by jmespadero** [comment, marked as answer]
Hi @JCAELA 

That question is not about a bug that developpers should fix, so it is better suitable to the section [Discussions](https://github.com/cnr-isti-vclab/PyMeshLab/discussions) than here. 

Said that, I'm quite proud of this [piece of code](https://stackoverflow.com/questions/65419221/how-to-use-pymeshlab-to-reduce-vertex-number-to-a-certain-number) that could help you. The arguments of simplification_quadric_edge_collapse_decimation and [meshing_decimation_quadric_edge_collapse_with_texture](https://pymeshlab.readthedocs.io/en/latest/filter_list.html#meshing_decimation_quadric_edge_collapse_with_texture) are almost the same.

**Comment by JCAELA** [comment]
Many thanks.

I really appreciated.
________________________________
De: jmespadero ***@***.***>
Enviado: segunda-feira, 4 de abril de 2022 11:30
Para: cnr-isti-vclab/PyMeshLab ***@***.***>
Cc: JCAELA ***@***.***>; Mention ***@***.***>
Assunto: Re: [cnr-isti-vclab/PyMeshLab] TEXTURED MESH SIMPLIFICATION (Issue #204)


Hi @JCAELA<https://github.com/JCAELA>

That question is not about a bug that developpers should fix, so it is better suitable to the section Discussions<https://github.com/cnr-isti-vclab/PyMeshLab/discussions> than here.

Said that, I'm quite proud of this piece of code<https://stackoverflow.com/questions/65419221/how-to-use-pymeshlab-to-reduce-vertex-number-to-a-certain-number> that could help you. The arguments of simplification_quadric_edge_collapse_decimation and meshing_decimation_quadric_edge_collapse_with_texture<https://pymeshlab.readthedocs.io/en/latest/filter_list.html#meshing_decimation_quadric_edge_collapse_with_texture> are almost the same.
